target/linux/mvebu/files/arch/arm/boot/dts/armada-385-linksys-rango.dts

```diff
@@ -440,6 +440,7 @@
         wlan_5g_led_pin: wlan-5g-led-pin {
                 marvell,pins = "mpp46";
 		marvell,function = "gpio";
+	};
```

Comment:
You have to add that brace to the patch which accidentally dropped it, not here.